jwst/tests_nightly/general/conftest.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import re
+import requests
+
+
+RE_URL = re.compile('\w+://\S+')
+
+
+def pytest_addoption(parser):
+    parser.addoption('--bigdata', action='store_true',
```

Comment:
As discussed offline, it might be a good idea to just have one `conftest` for all tests, regardless of Travis or Jenkins.

Reply:
Agreed. I'll likely consolidate this into the main `conftest` tonight or tomorrow.